docs/topics/wip_plugins.md

```diff
@@ -366,7 +364,7 @@ We will go from from top to bottom to find out what is happening and how to solv
 
 - **Welcome to HELL**
     
-    This is not a python issue, but as weird as it might sound, an implementation one.
+    This is not a python issue but, and as weird as it might sound, an implementation one.
```

Comment:
"but, and" doesn't sound natural remove the 'and' maybe

Reply:
Agreed